git/meta.yaml

```diff
@@ -1,16 +1,26 @@
 package:
   name: git
-  version: 2.4.6
+  version: 2.6.3
 
 source:
-  git_url: git@github.com:git/git.git
-  git_tag: v2.4.6
+  fn: v2.6.3.tar.gz
+  url: https://github.com/git/git/archive/v2.6.3.tar.gz
+  md5: 65f659fb55ba49f17de03d1d9a859a83
+
+build:
+  # git hardcodes paths to external utilities (e.g. curl)
+  detect_binary_files_with_prefix: true
 
 requirements:
   build:
+    - gcc
+    - expat
+    - curl
     - openssl
     - zlib
   run:
+    - expat
+    - curl
```

Comment:
`libgcc` is the name of the package we use to install just the dynamic libraries provided by gcc instead of the full `gcc` package (which is huge).

I think git is done in plain C, so there shouldn't be a problem when building with a new version of gcc and linking against an old one. Nevertheless, I'd prefer (for the sake of safety :-) to add `libgcc` as run dep here.